src/main/resources/application.properties

```diff
@@ -7,4 +7,4 @@ server.error.whitelabel.enabled=false
 server.error.path=/error
 spring-jpa.properties.hibernate.dialect=org.hibernate.dialect.PostgreSQLDialect
 spring-jpa.properties.hibernate.format_sql=true
-server.port=8080
+server.port=8085
```

Comment:
do not change.

Reply:
why?

Reply:
because this is a the standard port for spring boot. you should kill whatever is already running on 8080 instead of just randomly changing the port

Reply:
ah this was a miss understanding, I thought you don't want me to change it from 8085 to 8080, that's why I've asked "why", I have already changed it back to 8080